main.yml

```diff
@@ -352,8 +350,17 @@ settings:
   title: Cloud Services Settings
   frontend:
     sub_apps:
+      - id: ''
+        default: true
+        title: 'User Preferences'
       - id: hooks
+      - id: sources
+      - id: rbac
   top_level: true
+  isBeta: true
+  deployment_repo: https://github.com/RedHatInsights/settings-frontend-build
+  paths:
+  - /settings
 
 ruledev:
```

Comment:
I'm not sure why the `settings` app is above `ruledev`, but I can fix that after merge.